### PR TITLE
✨amp-ads: Added optional params for Directadvert network

### PR DESCRIPTION
**Pull request by dvdyakonov**
Added: 
- serverName
- canonicalUrl
- referrer

### CODE REVIEW

File: ads/directadvert.js [hunk 1]
```diff
@@ -23,10 +24,28 @@ import {loadScript, validateData} from '../3p/3p';
 export function directadvert(global, data) {
   validateData(data, ['blockId']);
```
Comment (calebcordry) [inline ads/directadvert.js]: Do you want to add server-name as an optional attribute here?

Comment (dvdyakonov) [reply ads/directadvert.js]: Is this necessary?

Comment (calebcordry) [reply ads/directadvert.js]: it is not required.

**Pull request merged**